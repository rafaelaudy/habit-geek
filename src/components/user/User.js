import React from "react";
import Login from "./Login";
import ProfileContainer from "./ProfileContainer";

const User = ({ jwt, saveSession }) => {
  return jwt ? <ProfileContainer /> : <Login saveSession={saveSession} />;
};
export default User;
