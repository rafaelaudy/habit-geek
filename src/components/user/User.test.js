import React from "react";
import { shallow } from "enzyme";
import User from "./User";

describe("User component", () => {
  it("Renders the profile component", () => {
    const user = shallow(<User jwt="jwt" />);
    expect(user).toMatchSnapshot();
  });

  it("Renders the login component", () => {
    const user = shallow(<User />);
    expect(user).toMatchSnapshot();
  });
});
