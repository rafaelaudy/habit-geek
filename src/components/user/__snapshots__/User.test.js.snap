// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User component Renders the login component 1`] = `<Login />`;

exports[`User component Renders the profile component 1`] = `<Connect(Profile) />`;
